changelogs/internal/newsfragments/1636.clarification

```diff
@@ -0,0 +1 @@
+Fix schema of `m.receipt` EDU.
```

Comment:
this isn't internal - it's part of the server-server api.

Reply:
The changelog was moved